rest/database/db.go

```diff
@@ -58,6 +58,9 @@ func Init() {
 	if !DB.Migrator().HasTable(&models.SelfReport{}) {
 		DB.Migrator().CreateTable(&models.SelfReport{})
 	}
+	if !DB.Migrator().HasTable(&models.DashboardTemplate{}) {
```

Comment:
Seeing errors using any of our user APIs as this table is not initialized.

Reply:
Have you run the migration script?

Reply:
I thought so... For now I've removed this change and will try again tomorrow.

Reply:
Yep - I think the migration script failed the first time I ran it. This works as expected without the change 👍